### DISCUSSION TITLE
Inconsistent transcriptions?

### DISCUSSION THREAD

**Discussion by RikRaesTNO** [openai/whisper#1312] [Q&A] [answered]
When using whisper (currently only the tiny version), I obtain different transcriptions every time I run them. I use the transcription using the code outlined below. Can anyone explain this or help me obtain consistent transcriptions?

`import whisper`
`model = whisper.load_model("tiny")`
`result = model.transcribe(audio)`
`text = result["text"]`

**Comment by phineas-pta** [comment, marked as answer]
it's the default behaviour see #81
